force-app/main/default/classes/Chat_SurveyHelperTest.cls

```diff
@@ -0,0 +1,22 @@
+@isTest
+public with sharing class Chat_SurveyHelperTest {
+    @isTest
+    static void testGetSurveyLink() {
+        Case enCase = new Case(Status = 'New');
+        LiveChatTranscript transcript = new LiveChatTranscript(
+            ChatKey = 'testKey',
+            CaseId = enCase.Id,
+            CRM_Authentication_Status__c = 'Completed',
+            NKS_Authentication_Timestamp__c = Datetime.now()
+        );
+        String surveyLink = '';
+
+        Test.startTest();
+        try {
+            surveyLink = Chat_SurveyHelper.getSurveyLink('testKey');
+        } catch (Exception e) {
+        }
+        Test.stopTest();
+        System.assertEquals('', surveyLink);
+    }
```

Comment:
Dette tester ingenting, det er bare for testdekning. Er det en grunn til at denne ikke tester funksjonaliteten?

Reply:
Her var det ikke mulig å teste faktisk funksjonaliteten. Opprettelse av surveyinvitation kjøres via flyt, og denne metoden skal bare hente invitasjon som allerede er generert. Selve opprettelsen er noe vanskelig å teste for survey må finnes plus en invitasjon relatert til undersøkelsen. Vi kan lage en survey for test i scratch men surveyInvitataion er ikke noe som kan retrieves fra org-en. Det eneste måte jeg fant var å lage en invitasjon i sandbox-er for at testen kjører uten problem.